[Bug]: package.json should export css files

### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (6) x64 Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
    Memory: 3.22 GB / 15.88 GB
  Browsers:
    Chrome: 111.0.5563.110
    Edge: Spartan (44.19041.1266.0), Chromium (111.0.1661.54)
    Internet Explorer: 11.0.19041.1566
```


### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/s/currying-pond-89zwg6?file=/src/App.tsx

### Bug Description

## Actual Behavior
Cannot import '@fluentui/react/dist/css/fabric.min.css', because it is not defined in package.json exports. This is not happening in codesandbox example, but is still an issue.

## Expected Behavior
I'm able to import css files directly into my project. It should work the same as with sass files.


### Logs

```shell
ERROR Module not found: [...] Error: Package path ./dist/css/fabric.min.css is not exported from package
```


### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

--- a/change/@fluentui-react-b3b9e7bb-d031-4766-b6f1-595b8c3892b9.json
+++ b/change/@fluentui-react-b3b9e7bb-d031-4766-b6f1-595b8c3892b9.json
@@ -1,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "chore: Exporting CSS files in @fluentui/react package.",
+  "packageName": "@fluentui/react",
+  "email": "Humberto.Morimoto@microsoft.com",
+  "dependentChangeType": "patch"
+}

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -70,6 +70,7 @@
     "react-dom": ">=16.8.0 <19.0.0"
   },
   "exports": {
+    "./dist/css/*": "./dist/css/*",
     "./dist/sass/*": "./dist/sass/*",
     "./lib/ActivityItem": {
       "types": "./lib/ActivityItem.d.ts",

